Data update (#2)
Imageomics/mimicry-madness: data/README.md

# Mimicry Data

This repository explores two different mimicry datasets:

1. [Coral Snake Mimicry Complex](#coral-snake-mimicry-complex)
2. [Moth-Wasp Mimics](#moth-wasp-mimics)

Note that the data and metadata associated with this repository are licensed differently from the code in this repository and from each other; licensing is as indicated with the dataset.

More details about the data associated to each project component are provided below.

## Coral Snake Mimicry Complex

### FUNED Mimic Data

> Images of all specimens in the FUNED natural history collection. These images were taken by Andressa Viol at the Fundação Ezequiel Dias. Each specimen image contains a 40% reflectance light UV standard and a color palette. The metadata file `metadata_FUNED_full.csv` contains the following columns:

Reviewer: a little pedantic but technically not all specimens in the FUNED collection, just the specimens that i was able to photograph. So perhaps change this to "Images of specimens of mimic or model species in the FUNED natural history collection", also to add clarification of what kinds of specimens

> Images of all specimens in the FUNED natural history collection. These images were taken by Andressa Viol at the Fundação Ezequiel Dias. Each specimen image contains a 40% reflectance light UV standard and a color palette. The metadata file `metadata_FUNED_full.csv` contains the following columns:

Author: ```suggestion
Images of specimens of coral snakes and their mimics from the FUNED natural history collection. These images were taken by Andressa Viol at the Fundação Ezequiel Dias. Each specimen image contains a 40% reflectance light UV standard and a color palette. The metadata file `metadata_FUNED_full.csv` contains the following columns:
```
Would this work?

- `image_date`: String. Date image was taken (`MM/DD/YYYY`).
- `notes`: String. Relevant notes about photography or specimen (not all full). May also included subspecies information where available (`venustissimus` is the only subspecies listed).

**License:** TBD

### Snake Mimic Toy Data

The metadata file `metadata_toy_dataset.csv` has a `mimicry_status` column which contains values `mimic`, `model`, `neither`, `bad mimic`.
These labels were added by Andressa Viol. 

Other column additions/modifications are:
- `View`: standardized to `Dorsal` or `Ventral`, where "Dorsal" indicates the image was taken of the snake's back, while "Ventral" means the image was taken with the snake lying belly-up.
- `filepath`: based on creating the embeddings from the repo data folder, so this is `images/snake_images/<Photo.File>_nobg.png`.
- `sci_name`: composed of `<Genus> <Species>` (columns in the original file).

**License:** The original metadata and associated images are from [Neotropical snake photographs](https://doi.org/10.7302/qta3-xs67) dataset[^1]. This data and associated metadata is [CC BY-NC](http://creativecommons.org/licenses/by-nc/4.0/).

## Moth-Wasp Mimics

> `mw-metadata.csv` and `mw-nobg-metadata.csv`: metadata associated to the moths, wasps, and flies imaged by Sol Carolina Parra Santos. 

Reviewer 2: I will put: by Sol Carolina Parra Santos from field collected specimens in Panama

> `mw-metadata.csv` and `mw-nobg-metadata.csv`: metadata associated to the moths, wasps, and flies imaged by Sol Carolina Parra Santos. 

Author: ```suggestion
`mw-metadata.csv` and `mw-nobg-metadata.csv`: metadata associated to the moths, wasps, and flies imaged and collected in Panama by Sol Carolina Parra Santos.
```
Is this good?
